modules/commandModule/tests/testCommandModule.py

```diff
@@ -0,0 +1,44 @@
+import random
+import json
+import math
+
+class mockCommandModule:
+    """
+    Mock Commmand Module that generates random POGI file for testing
+
+    ...
+
+    Attributes
+    ----------
+
+    pogiFileDirectory : str
+        string containing directory to POGI file
+    __pogiData : dict
+        dictionary of POGI data to be written to POGI file
+    __pogiData['gpsCoordinates'] : list
+        list containing, in order, random latitude, longitude, altitude
+    __pogiData['currentAirspeed] : list
+        list containing random airspeed in m/s
+    __pogiData['eulerAnglesOfPlane] : list
+        list containing, in order, random yaw, pitch, row angles of plane in radians
+    __pogiData['eulerAnglesOfCamera'] : list
+        list containing, in order, random yaw, pitch, row angles of camera in radians
+    __pogiData['isLanded'] : int
+        list containing either 1 for True or 0 for False to indicate whether drone has landed
+    """
+
+    def __init__(self, pogiFileDirectory: str):
```

Comment:
Ok, let's restructure this a little bit using the builder pattern. 

1) The constructor for this class should just set the pogiFileDirectory attribute. 
2) Need to break stuff up into functions. Define functions called get_gpsCoordinates, get_currentAirspeed, get_eulerAnglesofPlane, etc. for all the attributes of the pogi json. Each of these functions should return what is currently being assigned to each of these attributes. For example
`
get_gpsCoordinates():
   return {'latitude': random.uniform(-90, 90), 'longitude': random.uniform(-180, 180), 'altitude': random.uniform(0, 100)}
`
3) Define a function called build_pogi(). It should generate a dictionary and save it as a json using the ```write_to_json()``` function. self.__pogiData should no longer be a class attribute. It'll look something like:
```
build_json():
   pogi = {
      gpsCoordinates: get_gpsCoordinates(),
      currentAirspeed: get_currentAirspeed(),
      ....
   }
   write_to_json(pogi)
   return pogi
```